src/screens/Dashboard/types.ts

```diff
@@ -25,6 +25,7 @@ export type TripSearchParams = {
   fromLocation?: Location;
   toLocation?: Location;
   searchTime?: SearchTime;
+  callerRouteName?: any;
```

Comment:
@gorandalum I tried `string` here but seems like it doesn't accept it, according to the parameters it should be either object or undefined, but I saw in other places that are allowed to use string. 

NOTE: In theory, you can set string here but then the typescript complains in the trip search file.